src/components/common.events.js

```diff
@@ -31,6 +31,21 @@ export class CcSelectEvent extends CcEvent {
   }
 }
 
+/**
+ * Dispatched when a range selection changes.
+ * @extends {CcEvent<string[]>}
+ */
+export class CcRangeSelectEvent extends CcEvent {
+  static TYPE = 'cc-range-select';
+
+  /**
+   * @param {string[]} detail
+   */
+  constructor(detail) {
+    super(CcRangeSelectEvent.TYPE, detail);
+  }
+}
+
```

Comment:
question: I'm wondering why you put this one in the common events. Do you fill it will be used in some other components?